downgrade

```diff
@@ -35,6 +35,8 @@ EOF
 
 read_pacman_conf() { sed '/^#\?'"$1"' *= *\(.*\)$/!d; s//\1/' "$PACMAN_CONF"; }
 
+read_downgrade_conf() { grep -v '^\s*$\|^\s*\#' "$DOWNGRADE_ROOT_CONF" 2>/dev/null | xargs; }
```

Comment:
I came up with the following `grep` -- does it work? I find it easier to reason about, but it could be subjective.

```suggestion
read_downgrade_conf() { grep -E -v '^ *(#.*)?$')  "$DOWNGRADE_ROOT_CONF" 2>/dev/null | xargs; }
```

Reply:
This reads much better, thanks. There is a minor syntax error with an extra `)` next to the regular expression, but I will fix that with a rebase.